build/Targets/Imports.targets

```diff
@@ -329,7 +329,7 @@
   <!-- This is a copy of the Microsoft.VisualStudio.SDK.EmbedInteropTypes NuGet package, but only the list of
        assemblies that we need. The package includes things like EnvDTE which are reasonable for consumers, but
        strange since we actually _implement_ DTE and use it as an exchange type with generics in a few places. -->
-  <Target Name="LinkVSSDKEmbeddableAssemblies" DependsOnTargets="ResolveReferences" AfterTargets="ResolveReferences">
+  <Target Name="LinkVSSDKEmbeddableAssemblies" BeforeTargets="FindReferenceAssembliesForReferences" >
```

Comment:
I think you could do `BeforeTargets="FindReferenceAssembliesForReferences;ResolveReferences"` to resolve @davkean's concern.

It wouldn't be _wrong_ to specify a `DependsOnTargets` to ensure that `ReferencePath` is populated, @Pilchie, but it's not required because there's sufficient ordering in the targets--by the time `FindReferenceAssembliesForReferences` is considered, it's already populated.